scripts/smoke_test.sh

```diff
@@ -8,8 +8,13 @@ if [ ! -f ".env" ]; then
     cp .env.oss .env
 fi
 
-# start server in the background in development mode
-NODE_ENV=development nohup yarn start &
```

Comment:
did this end up blowing stuff up?

Reply:
Possibly; smoke tests just fail randomly ATM.

Reply:
ah. be right back, eating shoe 🚪